pipelines/rj_smtr/tasks.py

```diff
@@ -509,8 +509,9 @@ def bq_upload(
     partitions = {partitions}, type = {type(partitions)}
     """
     )
-    if status["error"] is not None:
-        return status["error"]
+    if status:
```

Comment:
@Hellcassius por que tem essa condicional?

Reply:
Para poder usar o bq_upload em um contexto que não a captura do SPPO, podendo usar ele conectado com outra task que não retorne um status necessariamente

Reply:
nao acho que faz sentido o input mudar, sempre tem que vir com `status`.

se teve qualquer erro numa task anterior tem que entrar nessa task, e ela sempre tem uma task anterior (porque o `load`tem que ser feito de algum `extract`,  seja uma API, FTP, etc)

Reply:
Isso implica em fazer um try/except na task anterior e possivelmente em todas as tasks upstream pra carregar esse erro, o que, pra mim, só faz sentido se for o caso de upar os logs de captura para algum outro uso, como é o caso do SPPO. Como não pretendemos desenvolver uma recaptura pra esse dado (o que falhar vamos rodar na mão) acho que faz sentido permitir que o extract só falhe normalmente sem carregar o erro

Reply:
No prefect 2.0, quando chegar, isso fica bem mais fácil por permitir declarar um flow como uma função python qualquer e usar condicionais como o python "natural", mas quando eu adicionei o catch em cada uma das tasks, pareceu desnecessário